src/components/Hosts/UpdateStatusModal.tsx

```diff
@@ -84,60 +79,42 @@ export const UpdateStatusModal: React.VFC<Props> = ({
   };
 
   return (
-    <Modal
+    <ConfirmationModal
       data-cy={dataCy}
-      visible={visible}
+      open={visible}
       onCancel={onClickCancel}
+      onSubmit={onClickUpdate}
       title="Update Host Status"
-      footer={[
-        <Button
-          key="modal-cancel-button"
-          data-cy="modal-cancel-button"
-          onClick={onClickCancel}
-        >
-          Cancel
-        </Button>,
-        <LoadingButton
-          key="modal-update-button"
-          data-cy="modal-update-button"
-          variant="primary"
-          disabled={!status}
-          loading={loadingUpdateHostStatus}
-          onClick={onClickUpdate}
-        >
-          Update
-        </LoadingButton>,
-      ]}
+      buttonText="Update"
+      submitDisabled={!status || loadingUpdateHostStatus}
     >
       <Body weight="medium">Host Status</Body>
-      <StyledSelect
+      <Select
         aria-labelledby="host-status-select"
         data-cy="host-status-select"
         value={status}
-        onChange={(s) => setHostStatus(s as UpdateHostStatus)}
+        onChange={(s) => {
+          setHostStatus(s as UpdateHostStatus);
+          console.log("SET BUTTON");
+        }}
       >
         {hostStatuses.map(({ title, value, key }) => (
           <Option key={key} value={value} data-cy={`${value}-option`}>
             {title}
           </Option>
         ))}
-      </StyledSelect>
+      </Select>
       <StyledTextArea
         label="Add Notes"
         data-cy="host-status-notes"
         value={notes}
         rows={6}
         onChange={(e) => setNotesValue(e.target.value)}
       />
-    </Modal>
+    </ConfirmationModal>
   );
 };
 
-// STYLES
-const StyledSelect = styled(Select)`
-  width: 100%;
-  margin-bottom: ${size.m};
-`;
 const StyledTextArea = styled(TextArea)`
   resize: none;
 `;
```

Comment:
[nit] This is already declared [in LeafyGreen](https://github.com/mongodb/leafygreen-ui/blob/a0d6638c42dac5ffb74955fdcfcc18bd6264bd0e/packages/text-area/src/styles.ts#LL21) so could be deleted.